cellular_automata/1D_custom_ruleset.py

```diff
@@ -0,0 +1,60 @@
+from PIL import Image
+
+
+#Creates the outputing image
+RESULT = Image.new('RGB', (31, 16))
```

Comment:
As discussed at https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md#coding-style variable names should be in lowercase.  All uppercase is reserved for constants in Python.